src/main/resources/org/jenkinsci/plugins/configfiles/ConfigFilesUI/setup-codemirror.js

```diff
@@ -0,0 +1,14 @@
+document.addEventListener("DOMContentLoaded", function() {
+    const textarea = document.querySelector("textarea#config\\.content");
+    const contentTypeElement = document.querySelector(".content-type-config");
+
+    const contentType = contentTypeElement.getAttribute("data-content-type");
```

Comment:
In other PRs for this project, we have been using `dataset` instead of `getAttribute`:

```suggestion
    const contentType = contentTypeElement.dataset.contentType;
```

While they are equivalent, I think we should be consistent throughout the project.